core/migrations/arangodb.py

```diff
@@ -0,0 +1,50 @@
+from core.database_arango import ArangoDatabase
+from core.migrations import migration
+
+
+class ArangoMigrationManager(migration.MigrationManager):
+    DB_TYPE = "arangodb"
+
+    def connect_to_db(self):
+        self.db = ArangoDatabase()
+        self.db.connect()
+
+        system_coll = self.db.collection("system")
```

Comment:
Reminder for later: we will have to update to async job all collection calls.

Reply:
done.